README.md

```diff
@@ -65,7 +57,42 @@ Clone the `predict-wildfires` repository locally. In a terminal, run:
 $ git clone https://github.com/IBM/predict-wildfires
 ```
 
-### Create Watson Studio services with IBM Cloud
+### 2. Create Watson Studio services with IBM Cloud
+
+* On [Watson Studio](https://dataplatform.cloud.ibm.com/) choose `New project`:
+
+![](https://github.com/IBM/pattern-images/blob/master/watson-studio/studio_choices.png)
+
+and pick `Data Science`:
+
+![](https://github.com/IBM/pattern-images/blob/master/watson-studio/project_choices.png)
+
+* Name the project and select your object storage:
+
+![](https://github.com/IBM/pattern-images/blob/master/watson-studio/new_project.png)
+
+* Choose `+ Add to project` -> `Data asset` or click the `01/10` button and add the data you dowloaded in the Prerequisite (MODIS_C6_*_7d.csv):
```

Comment:
s#01/10#10/01#
s/dowloaded/downloaded/
"in the Prerequisite" looks a little funny to me. Not sure exactly why but I think maybe either drop "the" and link or format it like it is a section or just say "the prerequisites" less formally (lower case).. or something like that